src/components/Header/index.js

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link';
+
+import SVG from '../../../public/SVG/svg';
+
+import StyledHeader from './styled';
+
+export default function Header() {
+	return (
+		<StyledHeader>
+			<h1>COFTEA</h1>
+			<Link href="./shoppingcart">
+				<a>
+					<SVG size="40px" color="lightgreen" variant="cart" />
+				</a>
+			</Link>
```

Comment:
I would probably add an `aria-label` here so that the `Link` has an accessible name.
```suggestion
			<Link href="./shoppingcart" aria-label="Shopping Cart">
				<a>
					<SVG size="40px" color="lightgreen" variant="cart" />
				</a>
			</Link>
```
Not sure though if it needs to be on the `Next.js` Link component or not 🤔